### PR TITLE
Column version for JDBC vendor type wrapping timestamp

### PR DESCRIPTION
**Pull request by cchantep**
Support type like `microsoft.sql.DateTimeOffset`: http://stackoverflow.com/questions/28716932/how-to-convert-microsoft-sql-datetimeoffset-to-date-in-anorm

### CODE REVIEW

**Comment by cchantep** [comment]
(Just first draft, doc and similar conversion for Joda and Java7 temporal types should be comited there too)

**Comment by cchantep** [comment]
Validated now

File: core/src/main/scala/anorm/ToStatement.scala [hunk 1]
```diff
@@ -53,8 +53,9 @@ object ToStatement extends JodaToStatement {
    * }}}
    */
   implicit object javaBooleanToStatement extends ToStatement[JBool] {
+    val jdbcType = implicitly[ParameterMetaData[JBool]].jdbcType
     def set(s: PreparedStatement, i: Int, b: JBool): Unit =
-      if (b != null) s.setBoolean(i, b) else s.setNull(i, Types.BOOLEAN)
+      if (b != null) s.setBoolean(i, b) else s.setNull(i, jdbcType)
```
Comment (cchantep) [inline core/src/main/scala/anorm/ToStatement.scala]: Uses meta typeclass to resolve the JDBC type

**Comment by jroper** [comment]
Nice.

**Comment by cchantep** [comment]
Thanks @jroper

**Pull request merged**